packages/pwa-buildpack/lib/queries/getAvailableStoreViews.graphql

```diff
@@ -0,0 +1,11 @@
+# Get available store views
+query {
+    availableStoreViews {
+        items {
+            code
+            name
+            locale
+            is_default
+        }
+    }
+}
```

Comment:
Your PR to Magento 2 didn't have this in the schema, but I was able to mock it out fine.

Reply:
There was a second PR for this I thought, one for StoreConfig and one for Translations. The second one should have had this in it.

Reply:
Looks like that PR never got submitted. My bad.